src/Utilities/ToolTask.cs

```diff
@@ -670,6 +670,7 @@ private string GetTemporaryResponseFile(string responseFileCommands, out string
             _standardOutputDataAvailable = new ManualResetEvent(false);
 
             _toolExited = new ManualResetEvent(false);
+            _terminatedTool = false;
```

Comment:
Are there any other things that need to be reset that currently aren't being reset? I'm wondering if we should move this to a separate "ResetToolTaskState" method.

Reply:
I've checked all related working variables (fields), I think all of them should be handled for now.